test/module/Entra/Get-EntraApplicationTemplate.Tests.ps1

```diff
@@ -0,0 +1,79 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+BeforeAll{
+    if ((Get-Module -Name Microsoft.Graph.Entra) -eq $null) {
+        Import-Module Microsoft.Graph.Entra
+    }
+    Import-Module (Join-Path $psscriptroot "..\Common-Functions.ps1") -Force
+
+    $response = @{
+        "id" = "aaaaaaaa-1111-2222-3333-cccccccccccc"
+        "supportedSingleSignOnModes" = @{}
+        "publisher" = "test publisher"
+        "displayName" = "test name"
+        "homePageUrl" = "samplehomePageUrl"
+        "logoUrl" = "samplelogourl"
+        "categories" = @{}
+        "description" = ""
+        "supportedProvisioningTypes" = @{}
+    }
+
+    Mock -CommandName Invoke-GraphRequest -MockWith { $response } -ModuleName Microsoft.Graph.Entra
+}
+Describe "Get-EntraApplicationTemplate tests"{
```

Comment:
Add tests for `Top` and `Property` params

Reply:
Done changes.

Reply:
added